dht-node/src/dht_node/index.ts

```diff
@@ -4,10 +4,21 @@ import DHT from '@hyperswarm/dht'
 import { logger } from '@/server/logger'
 import CONFIG from '@/config'
 import { FederatedCommunity as DbFederatedCommunity } from '@entity/FederatedCommunity'
+import { Community as DbCommunity } from '@entity/Community'
+import DEVOP from '@/config/devop'
+import { setDevOpEnvValue } from '@/config/tools'
+import { v4 as uuidv4 } from 'uuid'
 
 const KEY_SECRET_SEEDBYTES = 32
-const getSeed = (): Buffer | null =>
-  CONFIG.FEDERATION_DHT_SEED ? Buffer.alloc(KEY_SECRET_SEEDBYTES, CONFIG.FEDERATION_DHT_SEED) : null
+const getSeed = (): Buffer | null => {
+  let dhtseed = DEVOP.FEDERATION_DHT_SEED
+  logger.debug('dhtseed set by DEVOP.FEDERATION_DHT_SEED={}', DEVOP.FEDERATION_DHT_SEED)
+  if (!dhtseed) {
+    dhtseed = CONFIG.FEDERATION_DHT_SEED
+    logger.debug(`dhtseed overwritten by CONFIG.FEDERATION_DHT_SEED=${CONFIG.FEDERATION_DHT_SEED}`)
+  }
+  return dhtseed ? Buffer.alloc(KEY_SECRET_SEEDBYTES, dhtseed) : null
+}
```

Comment:
I do not think that the .devop mechanic is wise - that should be a subsystem on the deploy script (im not critisizing the .env.doevop itself - it is good to separate the generated secrets from the other .envs and deploy them to our processes)

Please make the whole thing in the deploy scripts not in the backend - since then we repeat all the code for all repos, invite new .env parsers etc - just make sure to propagate values from .env.devops to the existing .envs generated. That way no code change in any module but the the deployment is required.

If you do this please also do it for the other generated variables `JWT_SECRET` and `DB_PASSWORD`

Reply:
Yes thats the plan. the current state of implementation is the first working part from the dht-node and backend-modul point of view. For sure this must be extended to the deployment and good idea to add the other secrets in that file too.

Reply:
removed devop topic